src/components/VerifyEmail/VerifyEmail.tsx

```diff
@@ -1,19 +1,39 @@
+import {Typography} from '@mui/material'
 import {useMutation} from '@tanstack/react-query'
 import axios from 'axios'
 import {useRouter} from 'next/router'
 import {FC, useEffect} from 'react'
 
+import {Loading} from '../Loading/Loading'
+import {LoginForm} from '../PageLayout/LoginForm/LoginForm'
+
 export const VerifyEmail: FC = () => {
   const router = useRouter()
   const {verificationKey} = router.query
 
-  const {mutate: verifyEmail, isSuccess: isEmailVerified} = useMutation({
+  const {
+    mutate: verifyEmail,
+    isError: isError,
+    isSuccess: isVerified,
+  } = useMutation({
     mutationFn: (verificationKey: string) => axios.post('/api/user/registration/verify-email', {key: verificationKey}),
   })
 
   useEffect(() => {
     typeof verificationKey === 'string' && verifyEmail(verificationKey)
   }, [verificationKey, verifyEmail])
 
-  return <>isEmailVerified = {String(isEmailVerified)}</>
+  if (isError) return <Typography variant="body1">Email už bol verifikovaný, skús sa prihlásiť.</Typography>
```

Comment:
hmm ale padnut by to mohlo asi aj kvoli networku alebo hocicomu inemu... asi by som radsej dal aj tu moznost, nejako takto:
```
Email už bol verifikovaný, alebo nastal iný problém. Skús sa prihlásiť a v prípade problémov skús overiť email znovu alebo nás kontaktuj.
```